content/en/data_streams/manual_instrumentation.md

```diff
@@ -94,6 +94,69 @@ set_consume_checkpoint(
 "<datastream-type>", "<datastream-name>", headers.get
 )
 
+{{< /code-block >}}
+{{% /tab %}}
+{{% tab ".Net" %}}
+<div class="alert alert-warning">
+  <strong>Note:</strong> DSM instrumentation does not work in Async operations
+</div>
+
+In your producer
+{{< code-block lang="csharp" >}}
+using Datadog.Trace;
+
+using (var scope = Tracer.Instance.StartActive("produce"))
+{
+    var headers = new Headers();
+    var msg = new Message<Null, string> { Value = "any kind of value", Headers = headers};
```

Comment:
```suggestion
    var msg = new Message { Value = "any kind of value", Headers = headers};
```
Any specific reason to have template parameters here ? I feel like they are distracting from what's actually necessary in this example